server/controllers/lettersControllers/lettersControllers.js

```diff
@@ -40,9 +45,8 @@ exports.getAllLetters = async (req, res) => {
 
 exports.getLettersById = async (req, res) => {
     try {
-        const letterId = req.params.id
+        const {letterId} = req.params.id
         const letters = await Letters.findById(letterId).exec()
-        res.json(letters)
```

Comment:
When we reverted back to this commit, the response was removed. Let's add that back.